.circleci/config.yaml

```diff
@@ -0,0 +1,25 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version
+      - image: circleci/golang:1.10
```

Comment:
Will this select the latest Go 1.10.x?

Reply:
I don't know, and I don't care at the moment. My intent is to disable builds as soon as the project is setup.